pkg/agent/route/route_linux.go

```diff
@@ -731,6 +817,20 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet,
 			}...)
 		}
 	}
+	if c.proxyAll {
+		writeLine(iptablesData, []string{
+			"-A", antreaPreRoutingChain,
+			"-m", "comment", "--comment", `"Antrea: do not track external to external IP request packets"`,
```

Comment:
please add comments for why they shouldn't be tracked

Reply:
Will add.

Reply:
Sonw